packages/lesswrong/components/posts/PostsPage/PostBodyPrefix.tsx

```diff
@@ -34,7 +41,17 @@ const PostBodyPrefix = ({post, query, classes}: {
     </div>}
 
     <AlignmentCrosspostMessage post={post} />
-    { post.authorIsUnreviewed && !post.draft && <div className={classes.contentNotice}>This post is awaiting moderator approval</div>}
+    { post.authorIsUnreviewed && !post.draft && <div className={classes.contentNotice}>
```

Comment:
Can you add a comment for the EA Forum?
```suggestion
    {/* ea-forum-look-here */}
    { post.authorIsUnreviewed && !post.draft && <div className={classes.contentNotice}>
```

Reply:
Yep, makes sense.